docs/install_guides/pypi-openvino-dev.md

```diff
@@ -4,18 +4,25 @@ Copyright © 2018-2021 Intel Corporation
 
-**The Developer Package Includes the Following Components Installed by Default:**
+The **developer package** includes the following components installed by default:
+
+| Component        | Console Script                                                                   | Description                                                                                                                                                                                                                                                                                                   |  
+|------------------|---------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Model Optimizer](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) | `mo` |**Model Optimizer** imports, converts, and optimizes models that were trained in popular frameworks to a format usable by Intel tools, especially the Inference Engine. <br>Popular frameworks include Caffe\*, TensorFlow\*, MXNet\*, and ONNX\*.                                               |
+| [Benchmark Tool](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_benchmark_tool_README.html)| `benchmark_app` | **Benchmark Application** allows you to estimate deep learning inference performance on supported devices for synchronous and asynchronous modes.                                              |
+| [Accuracy Checker](https://docs.openvinotoolkit.org/latest/omz_tools_accuracy_checker.html) and <br> [Annotation Converter](https://docs.openvinotoolkit.org/latest/omz_tools_accuracy_checker_annotation_converters.html) | `accuracy_check` <br> `convert_annotation` |**Accuracy Checker**  is a deep learning accuracy validation tool that allows you to collect accuracy metrics against popular datasets. The main advantages of the tool are the flexibility of configuration and an impressive set of supported datasets, preprocessing, postprocessing, and metrics. <br> **Annotation Converter** is a utility for offline conversion of datasets to the format suitable for metric evaluation used in Accuracy Checker.                                            |
+| [Post-Training Optimization Tool](https://docs.openvinotoolkit.org/latest/pot_README.html)| `pot` |**Post-Training Optimization Tool** allows you to optimize trained models with advanced capabilities, such as quantization and low-precision optimizations, without the need to retrain or fine-tune models.                                             |
```

Comment:
POT can be used (and actually quite often used) as a library or package through its API. So it is worth mentioning how to reach this API. 
Right now it is located in the `compression.api`. In the next release, we are going to rename it to `pot.api`. We cannot do it in the LTS. @MaximProshin, can comment more why it is so.

Reply:
@helena-intel , @ryanloney, @andrew-zaytsev  Should we  add description for component's module names and API, as Alex proposed? IMHO, it is rather developer guide info than install guide.

Reply:
@MaximProshin @AlexKoff88  why can't we change this in an LTS release? The goal of LTS should be to provide stable API experience for runtime. POT is a developer tool. If it is not impacting a production deployment, I don't see why this cannot be changed now.

Reply:
Added reference to POT API.